platforms/jvm/plugins-jvm-test-suite/src/main/java/org/gradle/api/plugins/JvmTestSuitePlugin.java

```diff
@@ -61,7 +60,7 @@
  */
 @Incubating
 public abstract class JvmTestSuitePlugin implements Plugin<Project> {
-    public static final String DEFAULT_TEST_SUITE_NAME = JavaPluginHelper.DEFAULT_TEST_SUITE_NAME;
+    public static final String DEFAULT_TEST_SUITE_NAME = "test";
```

Comment:
Note for the future:

This is an implementation detail of the Java Plugin. We shouldn't advertise this name on the JVM test suite plugin, which can be applied without the Java plugin